src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -475,9 +476,9 @@ def add_agent_manager(self, config_file=None, common_cfg=None, timeout=30):
         if config_file is None:
             config_file = self.get_config_file("daos", "agent")
         if common_cfg is None:
-            common_cfg = CommonConfig(
-                self.server_group, DaosAgentTransportCredentials())
-
+            agent_transport = DaosAgentTransportCredentials(self.workdir)
+            agent_transport.allow_insecure.value = self.agent_insecure_mode
```

Comment:
(pylint-no-member) Instance of 'TestWithServers' has no 'agent_insecure_mode' member